pydicom/tests/test_JPEG_LS_transfer_syntax.py

```diff
@@ -66,40 +66,46 @@ def teardown_method(self, method):
     @pytest.mark.skipif(numpy_handler is None, reason=numpy_missing_message)
     def test_read_mr_with_numpy(self):
         pydicom.config.image_handlers = [numpy_handler]
-        with pytest.raises((NotImplementedError, )) as e:
-            _ = self.jpeg_ls_lossless.pixel_array
-        assert re.match(
-            ".*No available image handler could decode this transfer "
-            "syntax (1.2.840.10008.1.2.4.80|"
-            "JPEG-LS Lossless Image Compression).*", str(e))
+        msg = (
+            r"Unable to decode pixel data with a transfer syntax UID of "
+            r"'1.2.840.10008.1.2.4.80' \(JPEG-LS Lossless Image Compression\) "
+            r"as there are no suitable pixel data handlers available."
+        )
+        with pytest.raises(NotImplementedError, match=msg):
+            self.jpeg_ls_lossless.pixel_array
 
     @pytest.mark.skipif(numpy_handler is None, reason=numpy_missing_message)
     def test_read_emri_with_numpy(self):
         pydicom.config.image_handlers = [numpy_handler]
-        with pytest.raises((NotImplementedError, )) as e:
-            _ = self.emri_jpeg_ls_lossless.pixel_array
-        assert re.match(
-            ".*No available image handler could decode this transfer "
-            "syntax (1.2.840.10008.1.2.4.80|"
-            "JPEG-LS Lossless Image Compression).*", str(e))
+        msg = (
+            r"Unable to decode pixel data with a transfer syntax UID of "
+            r"'1.2.840.10008.1.2.4.80' \(JPEG-LS Lossless Image Compression\) "
```

Comment:
I learned that unrecognized escape sequences where legal in Python until 3.6, and in 3.6 they cause a deprecation warning - so they have updated PEP8Speaks, obviously.